### DISCUSSION TITLE
Implementing VPC with Pulumi Golang SDK

### DISCUSSION THREAD

**Discussion by syacko-sote** [pulumi/pulumi#8365] [Pulumi Providers] [answered]
I want to create a VPC on AWS, and I can find an example or package. Any help pointing me in the right direction would be great. Today is my second day trying Pulumi.

I have looked at the following already and didn't see anything:
- https://github.com/pulumi/examples
- https://github.com/pulumi/pulumi-aws-quickstart-vpc/blob/main/examples/simple-go/main.go
- https://pkg.go.dev/github.com/pulumi/pulumi/sdk/v3/go/pulumi

Any help would be great and thank you in advance,
Scott-

**Comment by syacko-sote** [comment]
I think I found the VPC functions under the ec2 package. Please confirm if this is correct.

**Comment by EvanBoyle** [comment, marked as answer]
Full documentation for all properties on the AWS VPC resource can be found here: https://www.pulumi.com/registry/packages/aws/api-docs/ec2/vpc/

We have api docs for the AWS quickstart component here: https://www.pulumi.com/registry/packages/aws-quickstart-vpc/api-docs/vpc/

As you note, this example shows using the aws quickstart component in go: https://github.com/pulumi/pulumi-aws-quickstart-vpc/blob/main/examples/simple-go/main.go

**Comment by syacko-sote** [comment]
Thank you the example helps a lot.

Scott

On Mon, Nov 15, 2021 at 12:01 PM Evan Boyle ***@***.***>
wrote:

> Full documentation for all properties on the AWS VPC resource can be found
> here: https://www.pulumi.com/registry/packages/aws/api-docs/ec2/vpc/
>
> We have api docs for the AWS quickstart component here:
> https://www.pulumi.com/registry/packages/aws-quickstart-vpc/api-docs/vpc/
>
> As you note, this example shows using the aws quickstart component in go:
> https://github.com/pulumi/pulumi-aws-quickstart-vpc/blob/main/examples/simple-go/main.go
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/pulumi/pulumi/discussions/8365#discussioncomment-1646991>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQC5B7BZCWGUIKPHNTQAGJDUMFRKVANCNFSM5HOUFKSA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 

Scott Yacko
***@***.***
phone: +16504830648 / WhatsApp user
www.getsote.com
Sote, Swahili pronoun meaning "All of us."
